src/VirtualClient/VirtualClient.Actions/Sysbench/SysbenchClientExecutor.cs

```diff
@@ -242,5 +263,78 @@ private async Task RunTPCCWorkloadAsync(EventContext telemetryContext, Cancellat
                 }
             }
         }
+
+        private async Task TruncateMySQLDatabaseAsync(EventContext telemetryContext, CancellationToken cancellationToken)
+        {
+            string arguments = $"{this.packageDirectory}/truncate-tables.py --dbName {this.DatabaseName}";
```

Comment:
who authored the truncate-tables.py, is it already in the package? Why are we choosing to do it in the ClientExecutor instead of the MySqlServer configuraiton?